modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc

```diff
@@ -92,7 +92,7 @@ ownCloud provides two encryption types:
 | *User-Key:*
 a| Every user has their own private/public key pairs, and the private key is protected by the user’s password.
 
-IMPORTANT: User-Key encryption has been depreciated with xref:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
+IMPORTANT: User-Key encryption has been depreciated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```

Comment:
```suggestion
IMPORTANT: User-Key encryption has been deprecated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```

We could also fix the word depreciated => deprecated while we are here, if it does not cause anything else to explode.